src/lexer/nmodl.ll

```diff
@@ -475,7 +475,8 @@ ELSE                    {
                         }
 
 <LINE_MODE>\n   |
-<LINE_MODE>\r\n         {
+<LINE_MODE>\r\n |
+<LINE_MODE>\r           {
                             /** For title return string without new line character */
```

Comment:
Just out of curiosity: how does neuron/mod2c parse the title?

Reply:
the parser of mod2c is really different and accept \r everywhere